components/badge/badges.css

```diff
@@ -0,0 +1,88 @@
+@import url(../../default.css);
+@import url(../avatar/avatar.css);
+
+.badge-container {
+  position: relative;
+}
+
+.badge {
+  text-align: center;
+  background-color: var(--primary-color);
+  padding: 10px;
+  height: 5px;
+  width: 5px;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: -12.8px;
+  right: -12.8px;
+}
+
+.large-badge {
+  text-align: center;
+  background-color: var(--primary-color);
+  padding: 13px;
+  height: 10px;
+  width: 10px;
+  border-radius: 50%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: -16px;
+  right: -16px;
+}
```

Comment:
Hey, you can put the common properties like `text-align`, `bg-color`, `flex`, etc. in a separate class and apply that class to all the badges. Then in individual badge class, you only need to take care of the positioning and size of the badge. 
This will reduce repetition.

Rest all looks good!